client/app/shared/modals/model-edit/model-edit.component.ts

```diff
@@ -37,8 +39,7 @@ export class ModelEdit {
   }
 
   close () {
-    let closeButton: HTMLElement = document.querySelector('#model-edit [data-close]')
-    closeButton.click()
+    jQuery('#edit-modal').modal().modal('hide')
```

Comment:
You might not need `jQuery`: http://youmightnotneedjquery.com/

Reply:
You know, at some point copying the response from one conversation to another becomes tedious.